docker/go-1.12.x/Dockerfile

```diff
@@ -3,6 +3,6 @@
 #
 # Released under the MIT license.
 
-FROM karalabe/xgo-1.12
+FROM karalabe/xgo-1.12.5
 
```

Comment:
As of now go 1.12.6 is the latest release. Would be appreciated to see this updated accordingly

Reply:
Thanks, I've added 1.11.11 and 1.12.6 to bring this up-to-date.